app/src/main/java/com/architectcoders/aacboard/ui/common/PermissionRequester.kt

```diff
@@ -0,0 +1,27 @@
+package com.architectcoders.aacboard.ui.common
+
+import android.app.Activity
+import com.karumi.dexter.Dexter
+import com.karumi.dexter.listener.PermissionDeniedResponse
+import com.karumi.dexter.listener.PermissionGrantedResponse
+import com.karumi.dexter.listener.single.BasePermissionListener
+
+class PermissionRequester(private val activity: Activity, private val permission: String) {
+
+    fun request(continuation: (Boolean) -> Unit) {
```

Comment:
Does the callback function require a boolean value as parameter? We are not using it when calling request from MainNavHostActivity

Reply:
You're right! removed